docs/reference/index.qmd

```diff
@@ -4,14 +4,6 @@ title: API Reference
 
 Welcome to the Graph-RAG Python API reference. This is a reference for all graph-rag packages.
 
-## Integrations
-
-[langchain_graph_retriever](langchain_graph_retriever/index.qmd)
-
-## Base Packages
-
-[graph_retriever](graph_retriever/index.qmd)
-
-## Helpers
-
-[graph_rag_example_helpers](graph_rag_example_helpers/index.qmd)
+* [graph_retriever](graph_retriever/index.qmd) - Base package providing traversal functionality.
+* [langchain_graph_retriever](langchain_graph_retriever/index.qmd) - Traversal functionality integrated as a LangChain retriever supporting a variety of LangChain `VectorStore`s.
+* [graph_rag_example_helpers](graph_rag_example_helpers/index.qmd) - Helpers used in various examples and notebooks.
```

Comment:
what is the reason for this formatting change?  I don't think this page works well with everything so grouped together.

Reply:
nevermind on this comment... My quartodoc split config is causing issues and this page will be updated in another PR.

Reply:
When I looked it didn't have any content in each of the headings, so it felt very bare. Seemed like it was just functioning as a redirect, so figured a list would be a bit simpler.